addons/website/static/src/js/theme_preview_kanban.js

```diff
@@ -24,13 +24,15 @@ var ThemePreviewKanbanController = KanbanController.extend(ThemePreviewControlle
             href: '/',
             innerHTML: '<i class="fa fa-close"></i>',
         });
-        const smallBreadcumb = Object.assign(document.createElement('small'), {
-            className: 'mx-2 text-muted',
-            innerHTML: _lt("Don't worry, you can switch later."),
-        });
+        if (!this.initialState.context.hide_breadcrumb_message) {
+            const smallBreadcumb = Object.assign(document.createElement('small'), {
+                className: 'mx-2 text-muted',
+                innerHTML: _lt("Don't worry, you can switch later."),
+            });
+            this._controlPanelWrapper.el.querySelector('.o_cp_top li').appendChild(smallBreadcumb);
+        }
         this._controlPanelWrapper.el.querySelector('.o_cp_top .breadcrumb li.active').classList.add('text-black-75');
         this._controlPanelWrapper.el.querySelector('.o_cp_top').appendChild(websiteLink);
```

Comment:
we can move this one also with hide breadcrumb.
As well you come from settings, and so you see "Settings" to go back.
As well you come from configurator skip / pick a theme, and the fa-close redirect to '/'

no?

//cc @pierre-pde

Reply:
You want to hide the fa-close when coming from settings because we can go back to settings using the breadcrumb ?

Reply:
yes, and because fa-close will redirect to front end, what is not obvious when you come from settings...

Reply:
@pierre-pde that's just moving one line so it just need your approval

Reply:
Go for it ;) he will probably not say **no** to fix a bug in UI ;)

Reply:
then using 'hide_breadcrumb_message' as key in the context is not accurate... Maybe something like 'from_backend' is better.

Reply:
from_setting ?

Reply:
That's quite specifc to our case. But ok. The day we will use this option from somewhere else we can rename it.